source/Mlos.Python/mlos/Examples/RustcFlagOptimization/bevybench/Cargo.lock

```diff
@@ -0,0 +1,2602 @@
+# This file is automatically @generated by Cargo.
```

Comment:
Should probably remove this file and add a `.gitignore` rule to prevent future inclusions as well.